.docker/scripts/create-test-data.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env sh
+##
+# Create some example content for extension BDD tests.
+#
+set -e
+
+export CKAN_ACTION_URL=http://ckan:3000/api/action
+export CKAN_INI_FILE=/app/ckan/default/production.ini
+
+. /app/ckan/default/bin/activate \
+    && cd /app/ckan/default/src/ckan
+
+# We know the "admin" sysadmin account exists, so we'll use her API KEY to create further data
+export API_KEY=$(paster --plugin=ckan user admin -c ${CKAN_INI_FILE} | tr -d '\n' | sed -r 's/^(.*)apikey=(\S*)(.*)/\2/')
+
+##
+# BEGIN: Create a Salsa Digital organisation with test users for admin, editor and member
+#
+SALSA_ORG_NAME=salsa-digital
+SALSA_ORG_TITLE="Salsa Digital"
+
+echo "Creating test users for ${SALSA_ORG_TITLE} Organisation:"
+
+paster --plugin=ckan user add ckan_user email=ckan_user@localhost password=password -c ${CKAN_INI_FILE}
+paster --plugin=ckan user add salsa_admin email=salsa_admin@localhost password=password -c ${CKAN_INI_FILE}
```

Comment:
can we please get rid of `Salsa` everywhere and use generic `test_` prefix. Using vendor name in open source is a very bad practice.